client/lib/site/utils.js

```diff
@@ -186,5 +186,9 @@ export default {
 		}
 
 		return site.domain !== site.wpcom_url;
+	},
+
+	isModuleActive( site, moduleId ) {
+		return site.modules && site.modules.indexOf( moduleId ) > -1;
```

Comment:
IIRC, `site.options.active_modules` contains the same data and was introduced in an earlier version of Jetpack, so may have better compatibility.